packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -782,20 +783,25 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 	// It might be buggy, refer to the below issue
 	// https://github.com/laurent22/joplin/pull/3974#issuecomment-718936703
 	useEffect(() => {
-		function pointerInsideEditor(params: any) {
-			const x = params.x, y = params.y, isEditable = params.isEditable, inputFieldType = params.inputFieldType;
+		function pointerInsideEditor(params: ContextMenuParams) {
+			const x = params.x, y = params.y, isEditable = params.isEditable;
 			const elements = document.getElementsByClassName('codeMirrorEditor');
 
-			// inputFieldType: The input field type of CodeMirror is "textarea" so the inputFieldType = "none",
-			// and any single-line input above codeMirror has inputFieldType value according to the type of input e.g.(text = plainText, password = password, ...).
-			if (!elements.length || !isEditable || inputFieldType !== 'none') return null;
+			// Note: We can't check inputFieldType here. When spellcheck is enabled,
+			// params.inputFieldType is "none". When spellcheck is disabled,
+			// params.inputFieldType is "plainText". Thus, such a check would be inconsistent.
+			if (!elements.length || !isEditable) return null;
```

Comment:
This logic can still be improved. With the current checks, it's possible that a password field in a dialog could pass these checks.

One way to improve this would be to check the value of `event.target` in the browser (not the electron) `contextmenu` event and use that in the above check.